docs/upgrade.html

```diff
@@ -61,6 +61,11 @@ <h4><a id="upgrade_1_0_0" href="#upgrade_1_0_0">Upgrading from 0.8.x, 0.9.x, 0.1
         Similarly for the message format version.</li>
 </ol>
 
+<h5><a id="upgrade_110_notable" href="#upgrade_110_notable">Notable changes in 1.1.0</a></h5>
```

Comment:
Thinking about this once more, I think we should put this into `streams/upgrade-guide.html` instead -- it a smaller improvement that does not need to be listed in the notable changes for the overall release (not a critical fix).